integrations/katalon_integration_demo_project/codemagic.yaml

```diff
@@ -1,22 +1,13 @@
 workflows:
   native-android:
-    name: Native Android
-    instance_type: mac_pro
+    name: Integrations Katalon Android
+    instance_type: mac_mini_m1
```

Comment:
`docker` is not available on the M1 instances. Probably best to leave this as `mac_pro`.